ext/opentelemetry-ext-otlp/tests/test_otlp_trace_exporter.py

```diff
@@ -240,11 +248,17 @@ def test_translate_spans(self):
                                             ),
                                             span_id=int.to_bytes(2, 8, "big"),
                                             attributes=[
-                                                AttributeKeyValue(
-                                                    key="a", int_value=1
+                                                KeyValue(
+                                                    key="a",
+                                                    value=AnyValue(
+                                                        int_value=1
+                                                    ),
                                                 ),
-                                                AttributeKeyValue(
-                                                    key="b", bool_value=False
+                                                KeyValue(
```

Comment:
Any reason to change the test values from a bool to an int?

Reply:
fixed